src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/UpdateRecallServiceTest.kt

```diff
@@ -108,15 +87,16 @@ class UpdateRecallServiceTest {
   @Test
   fun `can update recall with all fields populated`() {
```

Comment:
Can this fun name indicate what is special here about documents, i.e. why for "all fields populated" we use "fullyPopulatedRecall**WithoutDocuments**"?